lib/ProductOpener/Products.pm

```diff
@@ -623,6 +623,25 @@ sub change_product_server_or_code($$$) {
 	}
 }
 
+
+=head2 compute_sort_key ( $product_ref )
+
+Compute sort keys that are stored in the MongoDB database and used to order results of queries.
+
+=head3 last_modified_t - date of last modification of the product page
+
+Used on the web site for facets pages, except the index page.
+
+=head3 sortkey - date of last modification, with obsolete products last and complete products first
+
+Used on the web site for index page.
+
+=head3 search_key - Popular and recent products
```

Comment:
I'm not sure the variable/field name is good, considering it will be exposed in the database and possibly CSV. Right now, I think that something like `popularity_score` might represent the field content more precisely.

Reply:
I was thinking a more generic name would allow us to change the formula later. Today it's mostly popularity (as in scans), but later we could include other things (like for instance availability in stores if we get data from stores someday). The current formula in the code is in fact quite experimental: it's just one possible way, with the data we have today. Then I want to populate it in the full database, see what kind of results we get for different searches etc. and possibly adapt the formula.

Reply:
ok I renamed it to popularity_key . if we change the formula too much, we can always give it a new name / compute a different key.  I prefer "key" instead of "score" because the formula I made tries to achieve a certain order, but there's absolutely no scale for the key.